test/unit/test_keyring_raw_aes.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
```

Comment:
We also need to have unit tests for the `generate_data_key`:


- [ ] raise GenerateKeyError if os.urandom fails
- [ ] input MUST NOT contain data encryption key
- [ ] output MUST contain data encryption key and EXACTLY ONE WRAPPING_KEY_GENERATED_DATA_KEY trace for the input keyring